src/components/TimeSince.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2018 - 2022 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import dayjs from 'dayjs'
+import styled from 'styled-components'
+
+interface TimeSinceProps {
+  timestamp: number
+  className?: string
+  faded?: boolean
+}
+
+const TimeSince = ({ timestamp, className }: TimeSinceProps) => {
+  return <div className={className}>{dayjs(timestamp).fromNow()}</div>
+}
```

Comment:
nit: It's a dump component, no need for the `return` statement! :)

```suggestion
const TimeSince = ({ timestamp, className }: TimeSinceProps) => (
  <div className={className}>{dayjs(timestamp).fromNow()}</div>
)
```

Reply:
We use `const A = () => { return ... }` 18 other times in the project, could you explain how they're different than this? Like this little one:

```
d823a185 src/pages/WalletManagement/CheckWordsPage.tsx (Ilias Trichopoulos 2022-01-04 17:02:25 +0100 245) const getAlphabeticallyOrderedList = (arr: string[]) => {
d823a185 src/pages/WalletManagement/CheckWordsPage.tsx (Ilias Trichopoulos 2022-01-04 17:02:25 +0100 246)   return arr.slice().sort()
d823a185 src/pages/WalletManagement/CheckWordsPage.tsx (Ilias Trichopoulos 2022-01-04 17:02:25 +0100 247) }
```

I'm building up my external eslintrc file so we can reduce these inconsistencies, I'm just trying to understand the particularities so we can avoid any discussion about code style like the Go people :muscle: